src/debby/sqlite.nim

```diff
@@ -284,7 +284,7 @@ proc query*[T](
     columnCount = sqlite3_column_count(statement)
     headerIndex: seq[int]
   for i in 0 ..< columnCount:
-    let columnName = sqlite3_column_name(statement, i)
+    let columnName = $sqlite3_column_name(statement, i)
```

Comment:
I think this does a copy from cstring to string. Is this needed?

Reply:
Reason this is done is due to it possibly becoming a compiler error in the future, instead of converting it to a string, could convert any usage of a string, to a cstring explicitly when needed

Reply:
mysql also does this, probably good enough for now